Module not found errors on windows

Windows user here! :neutral_face: 
While starting the webpack server by command- ` webpack-dev-server --inline --hot --progress`, I'm getting Module not found errors.

```
ERROR in ./src/style/index.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modules/css-loader/index.js in D:\learning\preact\my-app/src\style
 @ ./src/style/index.less 4:14-197 13:2-17:4 14:20-203

ERROR in ./src/style/index.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modules/style-loader/addStyles.js in D:\learning\preact\my-app/src\style
 @ ./src/style/index.less 7:13-71

ERROR in ./src/components/home/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/css-loader/index.js in D:\learning\preact\my-app/src\components\home
 @ ./src/components/home/style.less 4:14-206 13:2-17:4 14:20-212

ERROR in ./src/components/home/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/style-loader/addStyles.js in D:\learning\preact\my-app/src\components\home
 @ ./src/components/home/style.less 7:13-74

ERROR in ./src/components/profile/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/css-loader/index.js in D:\learning\preact\my-app/src\components\profile
 @ ./src/components/profile/style.less 4:14-206 13:2-17:4 14:20-212

ERROR in ./src/components/profile/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/style-loader/addStyles.js in D:\learning\preact\my-app/src\components\profile
 @ ./src/components/profile/style.less 7:13-74

ERROR in ./src/components/header/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/css-loader/index.js in D:\learning\preact\my-app/src\components\header
 @ ./src/components/header/style.less 4:14-206 13:2-17:4 14:20-212

ERROR in ./src/components/header/style.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../../node_modules/style-loader/addStyles.js in D:\learning\preact\my-app/src\components\header
 @ ./src/components/header/style.less 7:13-74
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 502 bytes [rendered]
        [0] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 502 bytes {0} [built]
webpack: bundle is now VALID.
```

Tried cleaning npm cache and running `npm install` again but no luck.
Any suggestions/ideas would be helpful.

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "less-loader": "^2.2.3",
     "mocha": "^2.4.5",
     "ncp": "^2.0.0",
+    "path": "^0.12.7",
     "phantomjs-prebuilt": "^2.1.7",
     "postcss-loader": "^0.9.1",
     "raw-loader": "^0.5.1",

--- a/webpack.config.babel.js
+++ b/webpack.config.babel.js
@@ -2,17 +2,18 @@ import webpack from 'webpack';
 import ExtractTextPlugin from 'extract-text-webpack-plugin';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
 import autoprefixer from 'autoprefixer';
+import path from 'path';
 
 const ENV = process.env.NODE_ENV || 'development';
 
 const CSS_MAPS = ENV!=='production';
 
 module.exports = {
-	context: `${__dirname}/src`,
+	context: path.resolve(__dirname, "src"),
 	entry: './index.js',
 
 	output: {
-		path: `${__dirname}/build`,
+		path: path.resolve(__dirname, "build"),
 		publicPath: '/',
 		filename: 'bundle.js'
 	},
@@ -20,13 +21,13 @@ module.exports = {
 	resolve: {
 		extensions: ['', '.jsx', '.js', '.json', '.less'],
 		modulesDirectories: [
-			`${__dirname}/src/lib`,
-			`${__dirname}/node_modules`,
+			path.resolve(__dirname, "src/lib"),
+			path.resolve(__dirname, "node_modules"),
 			'node_modules'
 		],
 		alias: {
-			components: `${__dirname}/src/components`,		// used for tests
-			style: `${__dirname}/src/style`,
+			components: path.resolve(__dirname, "src/components"),		// used for tests
+			style: path.resolve(__dirname, "src/style"),
 			'react': 'preact-compat',
 			'react-dom': 'preact-compat'
 		}

